### ISSUE TITLE
Prevent automatic refresh after a new build

### ISSUE THREAD

**Issue by lbrdt** [TeamPiped/Piped#1199]
### Official Instance

- [X] The bug is reproducable on the [official hosted instance](http://piped.kavin.rocks/) or is API related.

### Describe the bug

When a new build is finished, new static resources are generated. So when Piped tries to sync with the newly built resources, it refreshes the page automatically without warning. Sometimes when you're watching the video. There needs to be a warning prompt to allow user to refresh when it is convenient.

### To Reproduce

1. Visit a piped instance watch some videos for it to cache its resource in the browser.
2. Start a build wwith a new commit is merged
3. Go to the Piped instance
4. Try waiting a while or view a video
5. The page refreshes on it's own

### Expected behavior

A litle prompt to allow user to refresh at their convenience

### Logs/Errors

No logs

### Browser, and OS with Version.

Chromium Browsers
FIrefox Mobile

### Additional context

It happens on both desktop and mobile browser

**Comment by lima46zulu** [comment]
I had the same behavior on the feed page -- when that page was working. All the screenshots on my feed would load, then right away the page would reload as I was about to click on a video. It never happened to me while watching a video ... but it's still very annoying.

**Comment by lima46zulu** [comment]
@opusforlife2 That is not the current behaviour on iPad OS (Chrome). Sometimes, not all ways, when I go to piped the page first loads. Then it automatically reloads ... and will continue to reload every few seconds or so. In other words the site is unusable when this happens. I have never noticed the notification button (I don't have my iPad handy), but the page just reloads on its own.

**Comment by moriel5** [comment]
I was wondering why the first time every time I opened Piped after clearing cache and cookies (I had set this to automatically occur upon browser close), it would refresh after a few minutes, regardless of what I am doing (this often happens to me while watching a video, like just now, when it refreshed the page when I was 1 minute and 43 seconds into the video).

**Comment by moriel5** [comment]
@opusforlife2 I agree, also Telegram and WhatsApp so something similar, where they offer a static overlay "Update Now" button to refresh (no option to dismiss).
